src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesComputer/sidepanel2.jelly

```diff
@@ -0,0 +1,29 @@
+<!--
```

Comment:
This sidepanel is not referenced from anywhere

Reply:
This file still working, it overrides from log.jelly.